packages/dnb-eufemia/src/extensions/payment-card/icons/index.js

```diff
@@ -4,7 +4,7 @@ import BankAxept from './BankAxept'
 import DNB from './DNB'
 import Expired from './Clock'
```

Comment:
```suggestion
import { clock_medium as Expired } from '../../../icons'
```
We should delete the `Clock.tsx` file.